### ISSUE TITLE
Wiki request - How to clear specific values from your browser's local storage

### ISSUE THREAD

**Issue by QuincyLarson** [FreeCodeCamp/FreeCodeCamp#3396]

**Comment by soulchainer** [comment]
I was just to try to do it (before I saw this), because some people ask this many often and I end repeating myself. Just yesterday, I helped another user with problems with this. So, it would be handy to have a wiki page for this.
But then, I just discovered [Clear Your Browser's Local Storage](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Clear-Your-Browser's-Local-Storage) and think: «Should instead edit this». But well, I guess that's more appropiate to make a new one, yes.
Just one thing (maybe this last should be in another issue): the wiki related page I mentioned is **dangerous**.
Because it places people to **delete all** their localStorage... and doesn't tip that it should backup their code previously if they want to preserve it. Because, their code it's only stored there, right?
I think that's risky and should be warned.

After sending this, I wrote it. [Here](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/How-to-clear-specific-values-from-your-browser's-local-storage).
I did it for Firefox, because it's what I use and the browser I support. This is a wiki, so anybody can edit it and add other browsers options. Like I noted, there's another way to do that, but sites with so many storage data, like FCC, makes Firefox to hang out (I tried the more «easy» way to do it, for explain it in the wiki, but the browser hanged out every time, so better do the other way, I guess).
Anyways, I think I explained in a direct (at least, I tried) and straightforward way.
